src/main/frontend/db/query_react.cljs

```diff
@@ -112,17 +112,20 @@
          f)) query)))
 
 (defn react-query
-  [repo {:keys [query inputs] :as query'} query-opts]
+  [repo {:keys [query inputs rules] :as query'} query-opts]
```

Comment:
This fn change is the only one that could effect outside the query-dsl functionality. Since it's an additive change, it should not effect other querying functionality